Mage.Sets/src/mage/cards/o/OhabiCaleria.java

```diff
@@ -48,9 +38,9 @@ public OhabiCaleria(UUID ownerId, CardSetInfo setInfo) {
         this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new UntapAllDuringEachOtherPlayersUntapStepEffect(filter)));
 
         // Whenever an Archer you control deals damage to a creature, you may pay {2}. If you do, draw a card.
-        this.addAbility(new OhabiCaleriaTriggeredAbility(new DoIfCostPaid(
+        this.addAbility(new DealsDamageToACreatureAllTriggeredAbility(new DoIfCostPaid(
                 new DrawCardSourceControllerEffect(1), new GenericManaCost(2)
-        )));
+        ), false, filter2, SetTargetPointer.PERMANENT, false));
```

Comment:
This should be `SetTargetPointer.NONE`. `DrawCardSourceControllerEffect` is not a targetted effect.

Reply:
Done